src/components/slider/slider.js

```diff
@@ -237,9 +336,9 @@ function SliderDirective($$rAF, $window, $mdAria, $mdUtil, $mdConstant, $mdThemi
       }
 
       var changeAmount;
-      if (ev.keyCode === $mdConstant.KEY_CODE.LEFT_ARROW) {
+      if (vertical ? ev.keyCode === $mdConstant.KEY_CODE.DOWN_ARROW : ev.keyCode === $mdConstant.KEY_CODE.LEFT_ARROW) {
```

Comment:
@EladBezalel Instead of checking Vertical/Horizontal in the line above, just always associate DOWN/LEFT with negative and UP/RIGHT with positive.